### ISSUE TITLE
TypeScript: can't use File as type

### ISSUE THREAD

**Issue by sangxxh** [nftstorage/nft.storage#1810]
## Problem

Using TypeScript, the type for `File` is not proper.

When using `File` as type, TypeScript shows this error:

```
'File' refers to a value, but is being used as a type here. Did you mean 'typeof File'?
```

It has to be written as `typeof File`, eg:

```
const files: typeof File[] = [];
```

`File` is a class, should developer should be able to use it as type instead of using `typeof File`

## Reproduce

```ts
import { NFTStorage, File } from 'nft.storage'
const client = new NFTStorage({ token: 'API_TOKEN' })

async function main() {
  // observe TypeScript error here <----------------------------------------------
  const image: File = new File(
    [
      /* data */
    ],
    'pinpie.jpg',
    { type: 'image/jpg' }
  );

  const metadata = await client.store({
    name: 'Pinpie',
    description: 'Pin is not delicious beef!',
    image,
  })
  console.log(metadata.url)
  // ipfs://bafyreib4pff766vhpbxbhjbqqnsh5emeznvujayjj4z2iu533cprgbz23m/metadata.json
}

main()
```

**Comment by alanshaw** [comment]
Ok @Gozala is going to take a look at this.

The temporary workaround that removed the error for me is to add "dom" to your lib: `"lib": ["es2021", "dom"]` in "compilerOptions".

**Comment by Gozala** [comment]
I spend quite some time digging into this but can't seem to figure out a way to fix this in the package itself. I do however noticed that just removing unnecessary type annotation addresses the problem, specifically following modification of the code works as expected:

```js
import { NFTStorage, File } from 'nft.storage'

const client = new NFTStorage({ token: 'API_TOKEN' })

async function main() {
  // observe TypeScript error here <----------------------------------------------
  const image = new File(
    [
      /* data */
    ],
    'pinpie.jpg',
    { type: 'image/jpg' }
  )

  const metadata = await client.store({
    name: 'Pinpie',
    description: 'Pin is not delicious beef!',
    image,
  })
  console.log(metadata.url)
  // ipfs://bafyreib4pff766vhpbxbhjbqqnsh5emeznvujayjj4z2iu533cprgbz23m/metadata.json
}

main()
```

**Comment by Gozala** [comment]
Should be fixed by #1915
